packages/lesswrong/components/review/ReviewVotingPage.tsx

```diff
@@ -171,24 +173,9 @@ const styles = (theme: ThemeType): JssStyles => ({
   }
 });
 
-export type ReviewVote = {_id: string, postId: string, score: number, type?: string}
-export type quadraticVote = ReviewVote & {type: "quadratic"}
-export type qualitativeVote = ReviewVote & {type: "qualitative", score: 0|1|2|3|4}
-
-
-const generatePermutation = (count: number, user: UsersCurrent|null): Array<number> => {
-  const seed = user?._id || "";
-  const rng = seedrandom(seed);
-  
-  let remaining = _.range(count);
-  let result: Array<number> = [];
-  while(remaining.length > 0) {
-    let idx = Math.floor(rng() * remaining.length);
-    result.push(remaining[idx]);
-    remaining.splice(idx, 1);
-  }
-  return result;
-}
+export type ReviewVotePart = {postId: string, score: number, type: 'QUALITATIVE' | 'QUADRATIC'}
```

Comment:
I'm a bit confused about the "Part" naming schema. In particular for "ReviewVotePart".

Reply:
I was thinking of renaming to SyntheticReviewVote. The important point is that it doesn't match the database type, which I'd be worried about people assuming.